client/components/mobile-phone-change/mobile-phone-change.js

```diff
@@ -140,14 +140,13 @@ class MobilePhoneChange extends React.Component {
       orgSlug,
       language,
       phone_number_change,
-      needsMobilePhoneVerification,
       settings
     } = this.props;
     const { input_fields } = phone_number_change;
     const { buttons } = phone_number_change;
 
     // check equality to false, it may be undefined
-    if (needsMobilePhoneVerification === false || !settings.mobile_phone_verification) {
```

Comment:
can you explain why this can be removed? Just to make sure I'm following you.

Reply:
This component can only be accessed if the user is authenticated. So it shouldn't be accessible only if `mobile_phone_verification` is true. 

- So if user is authenticated and mobile_phone_verification is true, user should be able to access this component and use it.
- if user is authenticated and mobile_phone_verification is false, user should not be able to access this component. hence he should be redirected